### ISSUE TITLE
Typescript error in Tooltip directive

### ISSUE THREAD

**Issue by ubau** [primefaces/primevue#943]
After upgrading to 3.2.x there is typescript error in Tooltip directive.
```
TS2693: 'Tooltip' only refers to a type, but is being used as a value here.

  > 34 | app.directive('tooltip', Tooltip);
       |                          ^^^^^^^
    35 | app.mount('#app');
    36 | 
```
Tooltip was ok in v.3.1.2

**Comment by avenmore** [comment]
Duplicate of [this issue](https://github.com/primefaces/primevue/issues/877).

**Comment by mertsincan** [comment]
Fixed in #877.

Best Regards,
